src/Driver/Http2Driver.php

```diff
@@ -1270,6 +1287,14 @@ private function parser(string $settings = null): \Generator
                             $windowSize = \unpack("N", $buffer)[1];
                             $buffer = \substr($buffer, 4);
 
+                            if ($windowSize < self::SMALL_WINDOW_UPDATE_THRESHOLD && ++$smallWindowUpdates > 10) {
```

Comment:
`$smallWindowUpdates` should be reset somewhere, otherwise it will definitely happen on a long living connection.

Reply:
Maybe this isn't necessary anyway, since many small window updates and small frames should be caught by the limits on total bytes vs. payload bytes. Let me give that some thought.